src/components/displayBox.css

```diff
@@ -0,0 +1,4 @@
+td, th{
+  padding:0px 4px;
+  border:1px solid;
```

Comment:
I would consider giving the table a different color.  I like black, makes it stand out.

![screen shot 2019-02-04 at 2 59 57 pm](https://user-images.githubusercontent.com/24882348/52233671-a7123980-288d-11e9-86b6-d5cea5f9a448.png)

Can do this by just adding a `color: black` to this css class.

Reply:
It looks like markdown does not support color - I know whatever renders it can have tables be a different color than regular text, but personally I think that would be unusual. Do we have any more information about how the EHR might render this?

Reply:
No our goal is to push this up as soon as possible so that Rush can look at it before HIMSS